documentation/src/main/asciidoc/cassandraclient.adoc

```diff
@@ -0,0 +1,301 @@
+= Quarkus - Using the Cassandra Client
+
+Cassandra is a well known NoSQL Database that is widely used.
+
+In this guide, we see how you can get your REST services to use the Cassandra database.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* GraalVM installed with `GRAALVM_HOME` configured appropriately if you want to use the native mode.
+* Apache Maven 3.5.3+
+* Cassandra installed or Docker installed
+
+== Architecture
+
+The application built in this guide is quite simple: the user can add elements in a list using a form and the list is updated.
+
+All the information between the browser and the server is formatted as JSON.
+
+The elements are stored in the Cassandra Database.
+
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+The solution is located in the `quickstart` {quickstart-clone-url}[directory].
+
+== Creating the Maven project
+
+First, we need a new project. Create a new maven project and copy the `pom.xml` file that is present in the `quickstart`.
+
+The `pom.xml` is importing the RESTEasy/JAX-RS, JSON-B, Context Propagation and Cassandra Client extensions.
+We will be building a REST application that is using DataStax Object Mapper. It will simplify the Data Access Layer code.
+
+The most important part of the `pom.xml` is adding the `cassandra-quarkus` extension:
+[source,xml]
+----
+<dependency>
+  <groupId>com.datastax.oss.quarkus</groupId>
+  <artifactId>cassandra-quarkus-client</artifactId>
+  <version>${project.version}</version>
```

Comment:
This isn't quite what we want, is it?  The user's project will shift versions as needed based on their development... I don't think we want to tie the version of the Quarkus extension to that.